src/utils/api.js

```diff
@@ -0,0 +1,32 @@
+import axios from "redaxios";
+
+const api = axios.create({
+  baseURL: "http://localhost:8080",
```

Comment:
It won't be a problem once deployed?

Reply:
Yes I pushed another version using dotenv files